src/app/query/non-fungible-tokens/non-fungible-token-holdings.hooks.ts

```diff
@@ -0,0 +1,23 @@
+import BigNumber from 'bignumber.js';
+
+import { SoftwareWalletAccountWithAddress } from '@app/store/accounts/account.models';
+
+import {
+  useGetNonFungibleTokenHoldingsListQuery,
+  useGetNonFungibleTokenHoldingsQuery,
+} from './non-fungible-token-holdings.query';
+
+export function useNonFungibleTokenHoldings(address: string | undefined) {
```

Comment:
```suggestion
export function useNonFungibleTokenHoldings(address?: string) {
```

Can also be expressed as an optional paramter